components/Changelog.js

```diff
@@ -6,6 +6,12 @@ const Changelog = React.createClass({
       <div>
         <h1>Change Log</h1>
 
+        <h3>4.2.21</h3>
+        <ul>
+          <li>Tabs Component - add it and make it work for mobile (<a href='https://github.com/mxenabled/mx-react-components/pull/395'>#395</a>)</li>
```

Comment:
The wording for this is odd.  How about something like `Adds new Tabs component.  See docs for details.`